docs/dev/contributing/backend/Guide-to-Creating-an-Endpoint.md

```diff
@@ -1,34 +1,207 @@
+                responses: response codes for this path, and a schema reference and the description if needed.
+                
+An example of the `Moves` path is as follows:
+
+      /moves:
+        get:
+          summary: listMoves
+          description: |
+            Gets all moves that have been reviewed and approved by the TOO. 
+          operationId: listMoves
+          tags:
+            - moveTaskOrder
+          produces:
+            - application/json
+          parameters:
+            - in: query
+              name: since
+              type: string
+              format: date-time
+              description: Only return moves updated since this time. Formatted like "2021-07-23T18:30:47.116Z"
+          responses:
+            '200':
+              description: Successfully retrieved moves. A successful fetch might still return zero moves.
+              schema:
+                $ref: '#/definitions/ListMoves'
+            '401':
+              $ref: 'responses/PermissionDenied.yaml'
+            '403':
+              $ref: 'responses/PermissionDenied.yaml'
+            '500':
+              $ref: '#/responses/ServerError'
+        
+#### Description Section and the response body
+In your endpoint description make sure that the following fields are included when necessary:
+* required - fields that are required are listed in the description section.
+* x-nullable - this indicates that the value of a particular property may be null. 
+* readOnly - sometimes you will need specify a readyonly property, for example when the property differs in a GET from a POST or PATCH. Note: readOnly properties are included in responses but not in requests.
+* eTag - An entity tag is provided so that a browser client or a script can make conditional REST requests using optimistic concurrency control. All eTags must be marked as readOnly.
+
+An example of the ListMove description is as follows:
+        
+          ListMove:
+            description: >
+              An abbreviated definition for a move, without all the nested information (shipments, service items, etc). Used to
+              fetch a list of moves more efficiently.
+            type: object
+            properties:
+              id:
+                example: 1f2270c7-7166-40ae-981e-b200ebdf3054
+                format: uuid
+                type: string
+              moveCode:
+                type: string
+                example: 'HYXFJF'
+                readOnly: true
+              createdAt:
+                format: date-time
+                type: string
+                readOnly: true
+              orderID:
+                example: c56a4180-65aa-42ec-a945-5fd21dec0538
+                format: uuid
+                type: string
+              referenceId:
+                example: 1001-3456
+                type: string
+              availableToPrimeAt:
+                format: date-time
+                type: string
+                x-nullable: true
```

Comment:
> It may be good to highlight that for any nullable field we need to use
> 
> ```
> x-nullable: true
> x-omitempty: false
> ```
> 
> This will return `null` in the response if the value doesn't exist. If we didn't all this, the field would be omitted.

I'm adding the field about the field being ommitted to this line on line 69:
* x-nullable - this indicates that the value of a particular property may be null.